File: kora-java-crud-submodule/kora-java-crud-submodule-vet-api/src/main/java/ru/tinkoff/kora/example/submodule/vet/VetModule.java
package ru.tinkoff.kora.example.submodule.vet;

import ru.tinkoff.kora.cache.caffeine.CaffeineCacheModule;
import ru.tinkoff.kora.common.KoraSubmodule;
import ru.tinkoff.kora.database.jdbc.JdbcDatabaseModule;
import ru.tinkoff.kora.example.submodule.common.CommonModule;
import ru.tinkoff.kora.resilient.ResilientModule;

@KoraSubmodule
public interface VetModule extends
        CommonModule,
        JdbcDatabaseModule,
        CaffeineCacheModule,
        ResilientModule {

}

File: examples/kotlin/kora-kotlin-crud-submodule/kora-kotlin-crud-submodule-vet-api/src/main/kotlin/ru/tinkoff/kora/kotlin/example/submodule/vet/VetModule.kt
package ru.tinkoff.kora.kotlin.example.submodule.vet

import ru.tinkoff.kora.cache.caffeine.CaffeineCacheModule
import ru.tinkoff.kora.common.KoraSubmodule
import ru.tinkoff.kora.database.jdbc.JdbcDatabaseModule
import ru.tinkoff.kora.kotlin.example.submodule.common.CommonModule
import ru.tinkoff.kora.resilient.ResilientModule

@KoraSubmodule
interface VetModule : CommonModule, JdbcDatabaseModule, CaffeineCacheModule, ResilientModule
